app/components/ui/Cards/OppEventCard.module.scss

```diff
@@ -0,0 +1,90 @@
+@import "../../../styles/utils/_helpers.scss";
+
+.card {
+  display: flex;
+  background: $white;
+  border: 2px solid $color-grey1;
+  border-radius: $rounded-md;
+  width: 100%;
+  overflow: hidden;
+  position: relative;
+  cursor: pointer;
+
+  &.opportunity {
+    flex-direction: column;
+  }
+
+  @media screen and (max-width: $break-tablet-l) {
+    flex-direction: column;
+  }
+
+  .cardImage {
+    height: auto;
+    width: 100%;
+    object-fit: cover;
+
+    &.event {
+      max-width: 40%;
+
+      @media screen and (max-width: $break-tablet-l) {
+        max-width: none;
+      }
+    }
+  }
+
+  .content {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    gap: $spacing-4;
+    padding: $spacing-6;
+    height: 100%;
+    width: 100%;
+
+    @media screen and (max-width: $break-tablet-p) {
+      padding: $spacing-4;
+    }
+
+    .contentTitle {
+      line-height: 125%;
+      margin-bottom: $spacing-4;
```

Comment:
can we use the $general-spacing-md here? I'd love to use those general spacing vars as much as possible!

Reply:
thanks for pointing that out!

Reply:
thank you!